src/main/java/com/sk89q/craftbook/core/LanguageManager.java

```diff
@@ -261,5 +261,6 @@ public static String getPlayersLanguage(Player p) {
         put("worldedit.ic.unsupported","WorldEdit selection type currently unsupported for IC's!");
         put("worldedit.ic.notfound","WorldEdit not found!");
         put("worldedit.ic.noselection","No selection was found!");
+        put("worldedit.ic.create","You've created");
```

Comment:
This is not at all WorldEdit related so it should not be under the worldedit namespace

Reply:
Oops, looks like i had a smooth brain moment. Lemme fix that... i also tested it and it works now.
![image](https://user-images.githubusercontent.com/24834067/151671204-871ae2c5-04a1-40bc-9d29-fc5a6122a17b.png)

*I wasnt sure if it should go under `mech.ic` or `circuits.ic`, but since the permission nodes for ICs are under `mechanics.ICs` i went for `mech.ic`*